linera-views/Cargo.toml

```diff
@@ -10,29 +10,30 @@ documentation = "https://docs.rs/linera-views/latest/linera_views/"
 license = "Apache-2.0"
 edition = "2021"
 
+# Not used, but here to provide the `js` feature to `rand`
+metadata.cargo-machete.ignored = ["getrandom"]
+
 [package.metadata.docs.rs]
-features = ["scylladb", "rocksdb", "aws", "test"]
+features = ["scylladb", "rocksdb", "dynamodb", "test"]
 targets = ["x86_64-unknown-linux-gnu", "wasm32-unknown-unknown"]
 
 [features]
-test = ["anyhow", "tokio/macros", "tokio/parking_lot"]
-aws = [
-    "aws-config",
-    "aws-sdk-dynamodb",
-    "aws-sdk-s3",
-    "aws-smithy-http",
-    "aws-smithy-types",
-    "aws-types",
-]
+test = ["tokio/macros", "rand"]
+metrics = ["dep:hex", "linera-base/metrics", "linera-views-derive/metrics"]
+
```

Comment:
Nit: unnecessary newline?

```suggestion
```

Reply:
This one's here to group the database features together :)

Reply:
Oh, right. I forgot `web` is going to be a database as well :sweat_smile:

Reply:
The database feature will be `indexeddb` like the others — but require `web`.